data/alexandria start.txt

```diff
@@ -0,0 +1,1304 @@
+			`	"Courier?  You're not a courier.  I don't understand."`
+			`	"You're paying me to ferry you around the galaxy looking for books."`
+			`	"No.  You don't understand.  You're my apprentice.  Alexandria Library doesn't ask just anyone to assist a Grandmaster Seeker.  They think you have potential to be a Seeker, despite all your faults.  The payments are just stipends to keep you fed during your apprenticeship."`
+			label "seeker questions"
+			choice
+				`	"What is a Seeker?"`
+					goto seeker
+				`	"Why me?"`
+					goto "why me"
+				`	"What faults?"`
+					goto faults
+				`	Change the subject...`
+					goto questions
+				`	"Sounds good.  Let's go."`
+					goto go
+				`	"I don't have time to help you any more."`
+					decline
+			label seeker
+			`"An Alexandria Seeker is an adventurer who scours the galaxy for works of literature or art and brings them back to the library for archival.  It's a dangerous job, but it is worth the risk to ensure knowledge is preserved.`
+			`	"When you get a pilot license, the paperwork goes to many organizations in the Republic.  Alexandria Library is one of them.  If your personality fits and you've show determination and resourcefulness, they consider you for an apprenticeship.  Finish your first mission, and you'll be a Journeyman Seeker.  Prove your skill, and you'll be a Master Seeker with a sizable salary.  We always Seek in pairs, so you'll still have to ferry someone even as a Master Seeker."`
+				goto "seeker questions"
+			label "why me"
+			`"I haven't read your file, but the reason is usually a mix of determination and resourcefulness based on your past five years of work.  There are basic educational requirements too, and a few other minor things."`
+				goto "seeker questions"
+			label faults
+			`"You must know yourself if you are to be a successful Seeker.  You are never truly strong until you turn your very weaknesses into strengths."`
+				goto "seeker questions"
+			label go
+			`"Excellent!  That's the spirit."`
+				accept
+	on decline
+		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
+		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
+	on fail
+		fail "Alexandria Library Card: Second Volume: Step 2 Escort"
+		fail "Alexandria Library Card: Second Volume: Step 2 Jump Instructions"
+	on visit
+		dialog `It seems you left the insane librarian or some of his thousand pencils behind.  Perhaps they're in one of your escorts?`
+	on complete
+		conversation
+			`Instead of landing on the main Amazon spaceport, Daemos has you land in a clearing deep inside one of the densest forests on the planet.  You climb through some aggressive vegetation, up a mountain, to a hut.  Daemos looks at you and says, "this is the place."`
+			choice
+				`	Open the door...`
+			`When the door opens, Daemos rushes forward in a panic, before you can even survey the room.  It seems the man you came to see is dead, and quite recently too.`
```

Comment:
I'm going to rewrite this entire encounter.  It was poorly done.

Reply:
An investigation doesn't make sense.  Daemos and the player would be in a hurry to get out rather than getting implicated in the crime.  I've modified the dialog so it is left as a mystery.